new-filename.js

```diff
@@ -0,0 +1,3 @@
+if(foo === "bar") {
+  console.log('something new for real2')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=5639)
```suggestion
  console.log("something new for real2")
```